lib/ruby_lsp/requests/support/rubocop_runner.rb

```diff
@@ -29,6 +29,7 @@ class ConfigurationError < StandardError; end
           [
             "--stderr", # Print any output to stderr so that our stdout does not get polluted
             "--force-exclusion",
+            "--raise-cop-error",
```

Comment:
We'll need to add a RuboCop version check and conditionally include this flag since it only exists after a certain version. Otherwise, it'll raise an error and complain about a non-existing flag.